### PR TITLE
R Birds Tutorial

### PR DESCRIPTION
**Pull request by cairosanders**
I work on the birds at PCIC that wrap R software ([quail](https://github.com/pacificclimate/quail) and [chickadee](https://github.com/pacificclimate/chickadee)) and I wanted to add some tips to the docs that might be helpful for others working on R birds.

### CODE REVIEW

**Comment by Zeitsperre** [comment]
@cairosanders Thanks for the PR! We'll give it a look soon!

File: docs/source/tutorial_r.rst [hunk 1]
```diff
@@ -0,0 +1,106 @@
+Pywps with R
```
Comment (huard) [inline docs/source/tutorial_r.rst:1]:
```suggestion
PyWPS with R
```

File: docs/source/tutorial_r.rst [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+============
+The following shows how you can wrap R software with pyWPS.
```
Comment (huard) [inline docs/source/tutorial_r.rst:3]:
```suggestion
The following shows how you can wrap R software with PyWPS.
```

File: docs/source/tutorial_r.rst [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+.. code-block:: Python
+
+   utils.install_packages("climdex.pcic")
+   climdex_pcic = importr("climdex.pcic")
+   climdex.climdex_gsl(climdexInput, gsl_mode)
+   
+.. _io:
+
+I/O
```
Comment (huard) [inline docs/source/tutorial_r.rst:59]: Any particular issue with LiteralInputs ? E.g. datetime objects.

**Review by huard — APPROVED** [review]

**Review by cehbrecht — APPROVED** [review]

**Comment by cehbrecht** [comment]
@cairosanders Looks good. Thanks :)

Please add it to the tutorials index:
https://github.com/bird-house/birdhouse-docs/blob/master/docs/source/tutorials.rst

I have fixed the bibtex on master ... `make docs` hopefully works again. If you like you can merge master and make a local check.

File: docs/source/tutorial_r.rst (hunk 3 above)
Comment (cairosanders) [reply docs/source/tutorial_r.rst:59]: From my experience, Rpy2 handles the conversion, so they can just be passed regularly without additional handling.

**Comment by cairosanders** [comment]
@huard, I'm just re-requesting review for the added note on LITERAL_DATA_TYPES to address your previous comment.

**Review by huard — APPROVED** [review]

**Comment by cehbrecht** [comment]
@cairosanders ready to merge?

**Comment by cairosanders** [comment]
@cehbrecht yes, thanks!

**Pull request merged**